src/sql/src/parsers/alter_parser.rs

```diff
@@ -434,6 +452,34 @@ fn parse_alter_column_drop_default(
     }
 }
 
+// Support parse multiple `SET DEFAULT`
+fn parse_alter_column_set_default(
+    parser: &mut Parser,
+) -> std::result::Result<SetDefaultsOperation, ParserError> {
+    parser.expect_keywords(&[Keyword::MODIFY, Keyword::COLUMN])?;
+    let column_name = ParserContext::canonicalize_identifier(parser.parse_identifier()?);
+    if parser.parse_keywords(&[Keyword::SET, Keyword::DEFAULT]) {
```

Comment:
We already parsed `modify column set default`, no need to parse them again. Also, the parser has parsed the column name.